client/landing/gutenboarding/onboarding-block/stepper-wizard.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+export interface StepInputProps {
+	isActive: boolean;
+	onExpand: () => void;
+	onSelect: () => void;
+	inputClass: string;
+}
+
+interface Props {
+	children: React.ReactChildren;
+}
+
+export default function StepperWizard( props: Props ) {
+	const [ activeStep, setActiveStep ] = useState( 0 );
+
+	const handleNext = () => setActiveStep( activeStep + 1 );
+
+	return React.Children.map(
+		props.children,
+		( child, index ) =>
+			child &&
+			React.cloneElement( child, {
```

Comment:
Getting a type error here. I found a relevant [SO thread](https://stackoverflow.com/questions/42261783/how-to-assign-the-correct-typing-to-react-cloneelement-when-giving-properties-to).

I was trying

```suggestion
			React.cloneElement( child as React.ReactElement< StepInputProps >, {
```

but was still getting a TS error (that I don't really understand :sweat_smile: ).

Maybe @sirreal can help? :grimacing: